core-dev/packages/coq/coq.dev/opam

```diff
@@ -22,6 +22,7 @@ depends: [
   "ocamlfind" {build}
   "num"
   "conf-findutils" {build}
+  "zarith"    { >= "1.9.1" }
```

Comment:
```suggestion
  "zarith" { >= "1.9.1" }
```

Reply:
Also, why version 1.9.1? The coq docs say ">= 1.8".

Reply:
Not sure why. I was copying from the opam file in coq/coq.